src/site/content/en/blog/share-like-a-native/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Share like a native app
```

Comment:
For discoverability, would it be good to include the API in the title? Like "Use the Web Share API to share like a native app"?

Reply:
```suggestion
title: Share like a native app with the Web Share API
```

I think so